alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/job/details/EmailJobDetailsModel.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.persistence.model.job.details;
+
+import java.util.List;
+
+public class EmailJobDetailsModel extends DistributionJobDetailsModel {
+    private final String subjectLine;
+    private final boolean projectOwnerOnly;
+    private final boolean additionalEmailAddressesOnly;
+    private final String attachmentFileType;
+    private final List<String> additionalEmailAddresses;
+
+    public EmailJobDetailsModel(String subjectLine, boolean projectOwnerOnly, boolean additionalEmailAddressesOnly, String attachmentFileType, List<String> additionalEmailAddresses) {
+        super("channel_email");
```

Comment:
We should get this key from the Email Channel Key. Either construct the Object here and call getUniversalKey() OR make this string public static final in the Email Channel Key.